src/interface/orthnull.jl

```diff
@@ -74,43 +78,47 @@ function left_orth(A::AbstractMatrix; kwargs...)
 
 Compute an orthonormal basis `V = adjoint(Vᴴ)` for the coimage of the matrix `A`, i.e.
 for the image of `adjoint(A)`, as well as a matrix `C` such that `A = C * Vᴴ`.
 The keyword argument `kind` can be used to specify the specific orthogonal decomposition
-that should be used to factor `A`, whereas `atol` and `rtol` can be used to control the
+that should be used to factor `A`, whereas `trunc` can be used to control the
 precision in determining the rank of `A` via its singular values.
 
+`trunc` can either be a truncation strategy object or a NamedTuple with fields
+`atol`, `rtol`, and `maxrank`.
+
 This is a high-level wrapper and will use call one of the decompositions
-`qr!`, `svd!`, and `left_polar!` to compute the orthogonal basis `V`, as controlled
-by the keyword arguments.
+`lq_compact!`, `svd_compact!`/`svd_trunc!`, and `right_polar!` to compute the
+orthogonal basis `V`, as controlled by the keyword arguments.
 
 When `kind` is provided, its possible values are
 
-*   `kind == :lqpos`: `C` and `Vᴴ` are computed using the positive QR decomposition.
-    This requires `iszero(atol) && iszero(rtol)` and `right_orth!(A, [CVᴴ])` is equivalent to
-    `lq_compact!(A, [CVᴴ], alg)` with a default value `alg = select_algorithm(lq_compact!, A; positive=true)`
-
 *   `kind == :lq`: `C` and `Vᴴ` are computed using the QR decomposition,
-    This requires `iszero(atol) && iszero(rtol)` and `right_orth!(A, [CVᴴ])` is equivalent to
-    `lq_compact!(A, [CVᴴ], alg)` with a default value `alg = select_algorithm(lq_compact!, A))`
+    This requires `isnothing(trunc)` and `right_orth!(A, [CVᴴ])` is equivalent to
+    `lq_compact!(A, [CVᴴ], alg)` with a default value `alg = select_algorithm(lq_compact!, A; positive=true)`
 
 *   `kind == :polar`: `C` and `Vᴴ` are computed using the polar decomposition,
-    This requires `iszero(atol) && iszero(rtol)` and `right_orth!(A, [CVᴴ])` is equivalent to
-    `right_polar!(A, [CVᴴ], alg)` with a default value `alg = select_algorithm(right_polar!, A))`
+    This requires `isnothing(trunc)` and `right_orth!(A, [CVᴴ])` is equivalent to
+    `right_polar!(A, [CVᴴ], alg)` with a default value `alg = select_algorithm(right_polar!, A)`
 
-*   `kind == :svd`: `C` and `Vᴴ` are computed using the singular value decomposition `svd_trunc!`,
-    where `V = adjoint(Vᴴ)` will contain the right singular vectors corresponding to the singular
-    values that are larger than `max(atol, rtol * σ₁)`, where `σ₁` is the largest singular value of `A`.
-    `C` is computed as the product of the singular values and the right singular vectors,
-    i.e. with `U, S, Vᴴ = svd_trunc!(A)`, we have `C = rmul!(U, S)` and `Vᴴ = Vᴴ`.
+*   `kind == :svd`: `C` and `Vᴴ` are computed using the singular value decomposition `svd_compact!`
+    if no truncation is specified through the `trunc` keyword argument or `svd_trunc!`
+    if truncation is specified through the `trunc` keyword argument.
```

Comment:
```markdown
`C` and `Vᴴ` are computed using the singular value decomposition `svd_trunc!` when a truncation strategy is specified using the `trunc` keyword argument, and using `svd_compact!` otherwise.
```